react/InstallmentsList.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useContext } from 'react'
+import { defineMessages } from 'react-intl'
+import { useCssHandles } from 'vtex.css-handles'
+import { ProductContext } from 'vtex.product-context'
+
+import { BasicPriceProps } from './types'
+import InstallmentsRenderer, {
+  Installments,
+} from './components/InstallmentsRenderer'
+
+const CSS_HANDLES = ['installmentsListContainer'] as const
+
+function InstallmentsList(props: BasicPriceProps) {
+  const { message, markers } = props
+  const { selectedItem } = useContext(ProductContext)
+  const handles = useCssHandles(CSS_HANDLES)
+
+  const commertialOffer = selectedItem?.sellers[0]?.commertialOffer
+
+  if (
+    !commertialOffer?.Installments ||
+    commertialOffer.Installments?.length === 0
+  ) {
+    return null
+  }
+
+  const sortedInstallments = commertialOffer.Installments.sort(
```

Comment:
`sort` mutates the original array.

https://stackoverflow.com/questions/9592740/how-can-you-sort-an-array-without-mutating-the-original-array